sdks/python/apache_beam/utils/dependency.py

```diff
@@ -139,7 +137,7 @@ def _stage_extra_packages(extra_packages,
       name patterns.
   """
   resources = []
-  tempdir = None
+  gcs_temp_dir = None
```

Comment:
Nit: can you please call this 'staging_temp_dir' or something like that. Calling it gcs_.... immediately makes me think that this is something that lives in GCS (i.e., remote storage not local).